### ISSUE TITLE
Not picking up paths of bower compenents where "main" points to root of repo

### ISSUE THREAD

**Issue by dkebler** [strarsis/sass-include-paths#5]
I wrote a bower only version of this using wiredep but thought I would try this as it's a bit more flexible.

But the bowerComponents function is only returning paths of repos whose "main" key points to a file in a subdirectory of the repo.

For example if bower install "bemiby"  it's main file is _bemify.scss in the root of that repo and your code does not add that to the array of paths.

Bower repos like "color-schemer" that have their "main" pointing to subdirectory like "/stylesheets" work fine.

here is my code. 

```
paths: function(data) {
    Debug.L1("in sass paths function")

    let sass_paths = []
    .concat(sassIncl.bowerComponentsSync());
    Debug.L1('paths:\n' +  sass_paths);
    return sass_paths;
````

**Comment by strarsis** [comment]
I use this code for the test:
````
var sassIncl = require('./');

var sass_paths = []
    .concat(sassIncl.bowerComponentsSync());
   console.log(sass_paths);
````
bemify and color-schemer have been installed using bower.


I get the following array:
````
[ 'bower_components/bemify/sass',
  'bower_components/color-schemer/stylesheets',
  'bower_components/scss-blend-modes/stylesheets' ]
````

The bemify/sass folder was added to the array.
Node v5.3.0, bower 1.7.9, color-schemer@0.2.8 and bemify@0.1.2 in this test.

**Comment by dkebler** [comment]
whoops sorry bemify was not the example with the issue it has main in a subdirectory and works fine  

try    "normalize-scss": "~3.0.3",
https://github.com/appleboy/normalize.scss

or sass-mq, or neutroncss or sass-flex-mixin...... any repo with main file in root of repo.

**Comment by strarsis** [comment]
Should be fixed now with release [v4.6.0](https://github.com/strarsis/sass-include-paths/releases/tag/v4.6.0).

**Comment by dkebler** [comment]
That did it...picks them all up now.  Thx
